src/current/v23.2/read-committed.md

```diff
@@ -0,0 +1,932 @@
+
+### Concurrency anomalies
+
+Statements in concurrent `READ COMMITTED` transactions can interleave with each other. This can create concurrency anomalies that are not permitted under `SERIALIZABLE` isolation, which places concurrent transactions into a [serializable ordering]({% link {{ page.version.version }}/demo-serializable.md %}).
+
+{{site.data.alerts.callout_success}}
+The behaviors described in this section assume the use of non-locking reads. You can prevent concurrency anomalies through the selective use of [locking reads](#locking-reads), which can also increase latency due to [lock contention]({% link {{ page.version.version }}/performance-best-practices-overview.md %}#transaction-contention).
+{{site.data.alerts.end}}
+
+#### Write skew anomaly
+
+The following sequence of operations is possible under `READ COMMITTED` isolation: 
+
+1. Transaction `A` reads row `R` in table `T` at timestamp `1`.
+1. Transaction `B` writes to row `R` in table `T` and commits at timestamp `2`.
+1. Transaction `A` writes to another row in table `T` and commits at timestamp `3`.
+
+The value of `R` has changed while transaction `A` is open. However, `A` can still write and commit instead of aborting, since `READ COMMITTED` transactions do not require serializability. This does not inherently create an anomaly, but is the basis of potential *write skew anomalies* where two concurrent transactions each read values that the other subsequently updates.
+
+{{site.data.alerts.callout_info}}
+For details on why this is allowed, see [Read refreshing]({% link {{ page.version.version }}/architecture/transaction-layer.md %}#write-skew-tolerance).
+{{site.data.alerts.end}}
+
+##### Example: Write skew anomaly
+
+For an example of how a write skew anomaly can occur, see [Demonstrate interleaved statements in `READ COMMITTED` transactions](#demonstrate-interleaved-statements-in-read-committed-transactions).
+
+#### Lost updates
+
+The `READ COMMITTED` conditions that permit [write skew anomalies](#write-skew-anomaly) also permit *lost updates*, where an update from a transaction appears to be "lost" because it is overwritten by a concurrent transaction:
```

Comment:
> The `READ COMMITTED` conditions that permit [write skew anomalies](#write-skew-anomaly) also permit

This isn't quite true. Lost updates aren't possible in snapshot isolation, but are in RC.

Using the terminology from the RFC, "write skew tolerance" is what permits writes skew anomalies and "per-statement read snapshots" is what permits lost update anomalies.